src/windows/functions.cpp

```diff
@@ -393,6 +393,38 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveEjectable(TCHAR driveLetter) {
+  TCHAR devicePath[8];
+  wsprintf(devicePath, TEXT("%c:\\"), driveLetter);
```

Comment:
Oh, damn. Can you open an issue for this? There is no way I'll be able to implement this and test it properly today. I'll handle it next week.